testutil/base.go

```diff
@@ -60,3 +62,31 @@ func (s *BaseTest) TearDownTest(c *check.C) {
 func (s *BaseTest) AddCleanup(f func()) {
 	s.cleanupHandlers = append(s.cleanupHandlers, f)
 }
+
+// Mocks any mutable object with a compliant substitute, and insures it will be restored after the test
```

Comment:
```suggestion
// Mocks any mutable object with a compliant substitute, and ensures it will be restored after the test
```

Reply:
Ooops, I did it again.

Thank you. :pray: